spec.html

```diff
@@ -21182,6 +21182,23 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
           1. Perform _classScope_.InitializeBinding(_classBinding_, _F_).
         1. Return _F_.
       </emu-alg>
+
+      <emu-clause id="sec-default-constructor-functions">
+        <h1>Default Constructor Functions</h1>
+        <p>When a Default Constructor Function is called with zero or more arguments which form the rest parameter ..._args_, the following steps are taken:</p>
+        <emu-alg>
+          1. If NewTarget is *undefined*, throw a *TypeError* exception.
+          1. Let _F_ be the active function object.
+          1. If _F_.[[ConstructorKind]] is ~derived~, then
+            1. NOTE: This branch behaves similarly to `constructor(...args) { super(...args); }`. The most notable distinction is that while the aforementioned ECMAScript source text observably calls `%Array.prototype[@@iterator]%`, this abstract operation does not.
+            1. Let _func_ be ? _F_.[[GetPrototypeOf]]().
+            1. If IsConstructor(_func_) is *false*, throw a *TypeError* exception.
+            1. Return ? Construct(_func_, _args_, NewTarget).
+          1. Else,
+            1. NOTE: This branch behaves similarly to `constructor() {}`.
+            1. Return ? OrdinaryCreateFromConstructor(NewTarget, `"%Object.prototype%"`).
+        </emu-alg>
+      </emu-clause>
```

Comment:
Note that this will need to be updated once the [**class fields proposal**](https://github.com/tc39/proposal-class-fields) reaches stage 4 and is merged.